cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,156 @@
 /// <reference types='cypress' />
+let worker
 
 describe('Web Tables page', () => {
-  it('', () => {
 
+  beforeEach(() => {
+    cy.task('newWorker').then(newWorker => {
+      worker = newWorker;
+   });
+    cy.visit('/');
+    
   });
+   
+  it('The page should contain the pagination', () => {
+    cy.get('.-pageInfo').should('contain.text', 'Page  of 1');
+    cy.contains('[disabled]', 'Previous').should('contain', 'Previous');
+    cy.contains('[disabled]', 'Next').should('contain', 'Next');
+    cy.get('[disabled]').should('contain.text', 'Next');
+    cy.get('[disabled]').should('contain.text', 'Previous');
```

Comment:
you do not check that after clicking next button, 2nd page is opened and after clicking previous button, previous page was opened